earthkit/transforms/aggregate/temporal.py

```diff
@@ -379,10 +379,13 @@ def daily_reduce(
         Name of the time dimension, or coordinate, in the xarray object,
         default behaviour is to deduce time dimension from
         attributes of coordinates, then fall back to `"time"`.
-    time_shift : (optional) timedelta or dict
+    time_shift : (optional) None, timedelta or dict
         A time shift to apply to the data prior to calculation, e.g. to change the local time zone.
         It can be provided as any object that can be understood by `pandas.Timedelta`, a dictonary is passed
-        as kwargs to `pandas.Timedelta`
+        as kwargs to `pandas.Timedelta`. Default is None.
+    remove_partial_periods : bool
+        If True and time_shift is not Null, the first and last time steps are removed to ensure equality in
```

Comment:
I would suggest a clarification of language
```suggestion
        If True and a time_shift has been applied, the first and last time steps are removed to ensure equality in
```

Reply:
done